app/views/layouts/_top_menu.html.erb

```diff
@@ -15,8 +15,9 @@
           <li class="dropdown">
             <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button">People<span class="caret"></span></a>
             <ul class="dropdown-menu">
-              <%= nav_link "Police", police_people_path %>
-              <%= nav_link "CERT", cert_people_path %>
+              <% Department.managing_people.order("shortname ASC").each do |dept| %>
+                <%= nav_link dept.shortname, department_people_path(dept) %>
```

Comment:
Not sure how my comment duplicated. 
I am not aware of any production deployments that don't have short name in the departments.
Seeds should be updated.
I really like the idea of the default shortname in the migration !